test/factories/prismaPool.factory.ts

```diff
@@ -2,78 +2,34 @@ import { Factory } from 'fishery';
 import { PrismaPoolWithDynamic } from '../../prisma/prisma-types';
 import { prismaPoolTokenFactory } from './prismaToken.factory';
 import { createRandomAddress } from '../utils';
+import { Chain, PrismaPoolType } from '@prisma/client';
+import { prismaPoolDynamicDataFactory } from './prismaPoolDynamicData.factory';
 
-export const prismaPoolFactory = Factory.define<PrismaPoolWithDynamic>(({ params }) => {
+class PrismaPoolFactory extends Factory<PrismaPoolWithDynamic> {
+    stable(amp?: string) {
+        return this.params({ type: PrismaPoolType.STABLE, typeData: { amp: amp ?? '10' } });
+    }
```

Comment:
@gmbronco - I added a variation of the pool factory so it creates a stable pool with proper parameters. How does that overlap with your new StablePoolFactory?

Reply:
Great, a stable pool factory would work. However, the point is more about design rather than functionality. The `prismaPool` is a database model, and SOR needs its own data format. Ideally, we should separate the SOR test so it doesn't depend on the database and can be worked on independently.